### PR TITLE
[7.7.x] RHPAM-1051: Stunner - some properties are not disabled in view mode 

### PR DESCRIPTION
**Pull request by wmedvede**
(cherry picked from commit 6a5c4a61c5830770046cd90d386bed36e83caef2)

### CODE REVIEW

**Comment by wmedvede** [comment]
Hello @romartin  @hasys , this is the peer PR for the already merged into master https://github.com/kiegroup/kie-wb-common/pull/1839. 
Would you mind review?
Thanks

**Comment by hasys** [comment]
Jenkins execute full downstream build

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-client/src/test/java/org/kie/workbench/common/stunner/bpmn/client/forms/fields/scriptEditor/ScriptTypeFieldRendererTest.java [hunk 1]
```diff
@@ -48,4 +48,6 @@ public void testGetSupportedCode() {
         assertEquals("ScriptTypeFieldType",
                      renderer.getSupportedCode());
     }
+
```
Comment (hasys) [inline kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-client/src/test/java/org/kie/workbench/common/stunner/bpmn/client/forms/fields/scriptEditor/ScriptTypeFieldRendererTest.java]: Is this a missing test or just useless new lines?

Comment (wmedvede) [reply kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-client/src/test/java/org/kie/workbench/common/stunner/bpmn/client/forms/fields/scriptEditor/ScriptTypeFieldRendererTest.java]: seems like a useless new line. I believe we can live with that.

**Comment by romartin** [comment]
Please @hasys can you give priority to this one? Let's see if we can include it for next `7.0.1`... thanks!!!

**Comment by domhanak** [comment]
@wmedvede Could you please submit PR's for both branches at once, next time? The approval process will be much faster.

**Comment by wmedvede** [comment]
@hasys thanks
@domhanak I always send the PR to the highest priority branch first and take a look how it evolves, because if there are changes/suggestions to apply it's a PITA to keep the PRs/branches in sync.

**Comment by wmedvede** [comment]
@romartin  I think this ready for going in for the 7.0.1

**Pull request merged**